### DISCUSSION TITLE
installed on arch, but can't run

### DISCUSSION THREAD

**Discussion by realmx2** [ClementTsang/bottom#1385] [Q&A] [answered]
Am I missing something. Installed in Arch system, but trying to run says it's not a valid command
```
[me@my-pc ~]$ bottom
bash: bottom: command not found
```
it says it's installed....
```
1 extra/bottom 0.9.6-1 (2.3 MiB 5.7 MiB) (Installed)
    A graphical process/system monitor
```

I've uninstalled and reinstalled, tried on another machine, etc. Am I missing something obvious?

**Comment by ClementTsang** [comment, marked as answer]
The command to run it is [`btm`](https://github.com/ClementTsang/bottom#usage).

**Reply by realmx2** [reply]
I figured it was something like that, but i couldn't find it in the documentation or help (but now I see it in "usage" and feel dumb). Thank you for your quick response (and this great program)

**Reply by kuolemaaa** [reply]
Encountered the same problem! Great application but I dont understand if there is a practical reason to not have named the binary `bottom`, is there? Any collision with something I am not aware of ?

**Reply by ClementTsang** [reply]
There's no real practical reason off the top of my head. I just wanted to give it a shorter name at the time. Typing out `bottom` every time to me back then felt annoying, so I settled on a shorter binary name. I guess I was influenced by how tools like `ripgrep` are `rg` as well?

I could theoretically make another binary with the full name but honestly that feels a little overkill for such a minor thing.
